public/app/plugins/datasource/tempo/resultTransformer.ts

```diff
@@ -161,51 +161,75 @@ function resourceToProcess(resource: collectorTypes.opentelemetryProto.resource.
   return { serviceName, serviceTags };
 }
 
-function getSpanTags(
-  span: collectorTypes.opentelemetryProto.trace.v1.Span,
-  instrumentationLibrary?: collectorTypes.opentelemetryProto.common.v1.InstrumentationLibrary
-): TraceKeyValuePair[] {
+function getSpanTags(span: collectorTypes.opentelemetryProto.trace.v1.Span): TraceKeyValuePair[] {
   const spanTags: TraceKeyValuePair[] = [];
 
-  if (instrumentationLibrary) {
-    if (instrumentationLibrary.name) {
-      spanTags.push({ key: 'otel.library.name', value: instrumentationLibrary.name });
-    }
-    if (instrumentationLibrary.version) {
-      spanTags.push({ key: 'otel.library.version', value: instrumentationLibrary.version });
-    }
-  }
-
   if (span.attributes) {
     for (const attribute of span.attributes) {
       spanTags.push({ key: attribute.key, value: getAttributeValue(attribute.value) });
     }
   }
 
+  return spanTags;
+}
+
+function getSpanKind(span: collectorTypes.opentelemetryProto.trace.v1.Span) {
+  let kind = undefined;
+  if (span.kind) {
+    const split = span.kind.toString().toLowerCase().split('_');
+    kind = split.length ? split[split.length - 1] : span.kind.toString();
+  }
+  return kind;
+}
+
+function getStatusCode(span: collectorTypes.opentelemetryProto.trace.v1.Span) {
+  let statusCode = undefined;
+  if (span.status) {
```

Comment:
Do we need these accessors here? Seems like they could be replaced by just using `?` operator like here `span.status?.code`?